src/data/constants.js

```diff
@@ -0,0 +1,70 @@
+import translationEn from "../i18n/en.json";
+import translationJa from "../i18n/ja.json";
+
+export const TIME_FORMAT = "MMMM d yyyy, HH:mm";
```

Comment:
Only used in one place, I suggest moving it to where it is actually used.

Reply:
But this is more of a configuration option than simply a string that is used, no?

Reply:
TIME_FORMAT is only used in one function

Reply:
That's fine, as @python273 also pointed out, its more of a configuration option, so moving it makes sense. Someone might fork this and 'time format' is a common thing that you may want to configure.